scripts/mbedtls_dev/build_tree.py

```diff
@@ -21,6 +22,26 @@ def looks_like_mbedtls_root(path: str) -> bool:
 def looks_like_root(path: str) -> bool:
     return looks_like_tf_psa_crypto_root(path) or looks_like_mbedtls_root(path)
 
+def crypto_core_directory(root: Optional[str] = None) -> str:
+    if root is None:
+        root = guess_project_root()
+    if looks_like_tf_psa_crypto_root(root):
+        return "core"
```

Comment:
The code only works when `root` is the current directory.

```suggestion
        return os.path.join(root, "core")
```

Same for `"library"` below.